docs/guides/dynamo_deploy/k8s_metrics.md

```diff
@@ -195,9 +213,18 @@ Visit http://localhost:9090 and try these example queries:
 
 ### In Grafana
 ```bash
-kubectl port-forward svc/grafana 3000:80 -n monitoring
+# Get Grafana credentials
```

Comment:
We can also point out, that the default password for this admin user is prom-operator